auraed/src/runtime/error.rs

```diff
@@ -2,27 +2,40 @@ use log::error;
 use thiserror::Error;
 use tonic::Status;
 
-#[derive(Error, Debug, PartialEq, Eq)]
-pub enum CellServiceError {
+pub(crate) type Result<T> = std::result::Result<T, RuntimeError>;
+
+#[derive(Error, Debug)]
+pub(crate) enum RuntimeError {
     #[error("missing argument in request: {arg}")]
     MissingArgument { arg: String },
 
+    #[error(transparent)]
+    ValidationError(#[from] validation::ValidationError),
+
     #[error("internal error: {msg}: {err}")]
     Internal { msg: String, err: String },
+
+    #[error(transparent)]
+    CgroupsError(#[from] cgroups_rs::error::Error),
+
+    #[error(transparent)]
+    Other(#[from] anyhow::Error),
 }
 
-impl From<CellServiceError> for Status {
-    fn from(err: CellServiceError) -> Self {
+impl From<RuntimeError> for Status {
```

Comment:
Looks like you were a step ahead of me, lol

Reply:
:D